src/main/main.ts

```diff
@@ -154,6 +150,7 @@ if (!app.requestSingleInstanceLock()) {
 
   cleanUp();
 
+  checkForUpdates(standalone);
```

Comment:
We should pass the `Update` returned from `checkForUpdates` to the `update` function. The result is unused here.